recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts

```diff
@@ -314,9 +354,39 @@
 };
 
 &a57_0 {
```

Comment:
will it be a good idea to put a comment why you only need a57 clocks, but not a53?

Reply:
Patch description already has explanation about it.
This patch adds the clock specifiers so that the CPU DVFS feature
can be enabled for A57 cores.
As A53 cores are not DVFS capable, remove existing non-SCPI clock specifiers
for them.

Reply:
while that being true it is sometimes easier to see a comment in the code
anyway, up to you